linode/rdns/framework_resource_schema.go

```diff
@@ -0,0 +1,47 @@
+package rdns
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/booldefault"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringplanmodifier"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/linode/terraform-provider-linode/linode/helper"
+)
+
+var frameworkResourceSchema = schema.Schema{
+	Attributes: map[string]schema.Attribute{
+		"address": schema.StringAttribute{
+			Description: "The public Linode IPv4 or IPv6 address to operate on.",
+			Required:    true,
+			PlanModifiers: []planmodifier.String{
+				stringplanmodifier.RequiresReplace(),
+			},
+			Validators: []validator.String{
+				helper.NewIPStringValidator(),
+			},
+		},
+		"rdns": schema.StringAttribute{
+			Description: "The reverse DNS assigned to this address. For public IPv4 addresses, this will be set " +
+				"to a default value provided by Linode if not explicitly set.",
+			Required: true,
+			Validators: []validator.String{
+				helper.NewStringLengthValidator(3, 254),
+			},
+			PlanModifiers: []planmodifier.String{
+				stringplanmodifier.UseStateForUnknown(),
+			},
+		},
+		"wait_for_available": schema.BoolAttribute{
+			Description: "If true, the RDNS assignment will be retried within the operation timeout period.",
+			Optional:    true,
+			Computed:    true,
```

Comment:
Can we default it to be true (or false) instead of saying it's a computed field? I am not sure if this is better though, but it seems a little bit more clear this way. So it's an optional suggestion.

Reply:
Unlike in SDKv2, Framework requires that all optional values with defaults also be computed:

> Attribute: 
                Summary: Schema Using Attribute Default For Non-Computed Attribute
                Detail: Attribute "wait_for_available" must be computed when using default. This is an issue with the provider and should be reported to the provider developers.

It's super weird, but I think this can stay as-is for now :slightly_smiling_face: